### DISCUSSION TITLE
Add Firefox version in 'About Floorp' page

### DISCUSSION THREAD

**Discussion by schembriaiden** [Floorp-Projects/Floorp#1236] [Feature Request]
I was thinking maybe the current Firefox version being used can be added next to the Floorp version in the about page:

![Screenshot 2024-06-22 154517](https://github.com/Floorp-Projects/Floorp/assets/81376423/a84c326c-7eff-4153-9dfd-de298ef34318)

**Comment by regularentropy** [comment]
![image](https://github.com/Floorp-Projects/Floorp/assets/89523758/a6f72b29-27d2-450c-a833-2b2b2a5cce07)
You can find it in the settings

**Comment by schembriaiden** [comment]
I know you can find it there, just thought it would be better to have it where I marked also.

**Comment by rguhr** [comment]
Looks like this was implemented.

<img width="688" height="485" alt="image" src="https://github.com/user-attachments/assets/03abd18d-1b0f-4eab-bd37-8e299bbedae9" />

**Comment by CutterKnife** [comment]
Closing this as completed. Thank you!
